Library/Homebrew/bottle_api.rb

```diff
@@ -71,6 +71,14 @@ def fetch_bottles(name)
     download_bottle(hash, bottle_tag)
 
     hash["dependencies"].each do |dep_hash|
+      existing_formula = begin
+        Formulary.factory dep_hash["name"]
+      rescue FormulaUnavailableError
+        nil
```

Comment:
A comment explaining why we don't care about it being unavailable would be good.